src/backend/src/services/work-packages.services.ts

```diff
@@ -447,8 +447,8 @@ export default class WorkPackagesService {
         wbsElement: {
           update: {
             name,
-            projectLeadId: projectLead,
-            projectManagerId: projectManager
+            projectLeadId: projectLeadId,
+            projectManagerId: projectManagerId
```

Comment:
you can make this line simply `projectManagerId`

Reply:
I think I did these now. Lmk if I have to change them or not.